minimalkv/stores/_net_common.py

```diff
@@ -1,3 +1,4 @@
+# TODO: Rename this module.
```

Comment:
This still needs to be done.

Reply:
Given that the size of this PR and that this is a private module that we could rename at any time, I suggest we don't change anything about this module in this PR.

```suggestion
```